### PR TITLE
Outdated CLI examples for start command - Closes #435

### LINKED ISSUE #435: Outdated CLI examples for start command
### Expected behavior

The examples given in command line must be accurate and up-to-date. 

### Actual behavior

Currently `lis-core start --help` shows following examples. 

```
EXAMPLES
  start
  start --network dev --data-path ./data --log debug
```
When we use following example it faild.

```
$ lisk-core start --network dev --data-path ./data --log debug

Starting Lisk Core at /Users/nazar/.lisk/lisk-core
 ›   Error: Network must be one of alphanet,betanet,devnet,mainnet,testnet but received dev
```

### Additional Information 
- There are many new features added to CLI, but the examples are very limited. We should think of adding more examples 
- Similar behaviour should be verified for other commands as well 

### Which version(s) does this affect? (Environment, OS, etc...)
3.0

### PR DESCRIPTION
**Pull request by nazarhussain**
### What was the problem?

This PR resolves #435

### How was it solved?

- Updated examples for different commands
- Updated some of the flags to use proper Kebab case

### How was it tested?

Run all tests

### CODE REVIEW

**Review by ManuGowda — APPROVED** [review]

File: src/commands/passphrase/encrypt.ts [hunk 1]
```diff
@@ -39,19 +39,27 @@ const processInputs = (passphrase: string, password: string, outputPublicKey: bo
 export default class EncryptCommand extends Command {
 	static description = 'Encrypt secret passphrase using password.';
 
-	static examples = ['passphrase:encrypt'];
+	static examples = [
+		'passphrase:encrypt',
+		'passphrase:encrypt --password your-password',
```
Comment (Incede) [inline src/commands/passphrase/encrypt.ts:44]: maybe include one more example `passphrase:encrypt --passphrase your-passphrase`

File: src/commands/passphrase/encrypt.ts [hunk 2]
```diff
@@ -39,19 +39,27 @@ const processInputs = (passphrase: string, password: string, outputPublicKey: bo
 export default class EncryptCommand extends Command {
 	static description = 'Encrypt secret passphrase using password.';
 
-	static examples = ['passphrase:encrypt'];
+	static examples = [
+		'passphrase:encrypt',
```
Comment (Incede) [inline src/commands/passphrase/encrypt.ts:43]: 'passphrase:encrypt --passphrase your-passphrase'

**Review by Incede — CHANGES_REQUESTED** [review]

File: src/commands/sdk/link.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ import { join, isAbsolute } from 'path';
 export default class LinkCommand extends Command {
 	static description = 'Symlink specific SDK folder during development.';
 
-	static examples = ['sdk:link /sdk/location/'];
+	static examples = ['sdk:link ~/lisk-sdk-rpo/sdk', 'sdk:link /sdk/location/'];
```
Comment (Incede) [inline src/commands/sdk/link.ts:24]: static examples = ['sdk:link /path/to/lisk-sdk/sdk'];
```suggestion
	static examples = ['sdk:link /path/to/lisk-sdk/sdk'];
```

File: src/commands/start.ts [hunk 4]
```diff
@@ -71,7 +71,16 @@ const setPluginConfig = (config: ApplicationConfig, flags: Flags): void => {
 export default class StartCommand extends Command {
 	static description = 'Start Lisk Core Node.';
 
-	static examples = ['start', 'start --network dev --data-path ./data --log debug'];
+	static examples = [
+		'start',
+		'start --network devnet --data-path ~/lisk-data --log debug',
```
Comment (Incede) [inline src/commands/start.ts:76]: 'start --network devnet --data-path /path/to/data-dir --log debug',
```suggestion
		'start --network devnet --data-path /path/to/data-dir --log debug',
```

**Review by Incede — APPROVED** [review]

**Pull request merged**